packages/material-ui/src/Tabs/Tabs.test.js

```diff
@@ -336,48 +338,52 @@ describe('<Tabs />', () => {
           <Tab />
         </Tabs>,
       );
+
       wrapper
         .find(Tab)
         .at(1)
         .simulate('click');
       wrapper.setProps({ value: 1 });
+
       assert.strictEqual(handleChange.callCount, 1, 'should have been called once');
       assert.strictEqual(handleChange.args[0][1], 1, 'should have been called with value 1');
-      wrapper.unmount();
```

Comment:
We need to clear the listeners. How do you solve the problem?
```
    this.handleResize.clear();
    this.handleTabsScroll.clear();
```

Reply:
I'm not really sure why I need to specifically unmount here. We're not doing it anywhere else. I introduced some unmounts in `afterEach` but it's still not obvious to if I should do this every time. I kind of thought mount is doing this by itself on cleanup.

Reply:
> I kind of thought mount is doing this by itself on cleanup.

I would expect that too. If it works without, it's perfect 👍.